tests/framework/utils/k8s_helper.go

```diff
@@ -528,14 +541,47 @@ func (k8sh *K8sHelper) waitForVolumeAttachment(namespace, volumeAttachmentName s
 	return fmt.Errorf("timeout for VolumeAttachment %s in namespace %s wait to %s", volumeAttachmentName, namespace, action)
 }
 
+func (k8sh *K8sHelper) printPVs() {
+	pvs, err := k8sh.Clientset.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
+	if err != nil {
+		logger.Infof("faied to list pvs. %+v", err)
+	}
+
+	var names []string
+	for _, pv := range pvs.Items {
+		names = append(names, pv.Name)
+	}
+	logger.Infof("Found PVs: %v", names)
+}
+
+func (k8sh *K8sHelper) printPVCs(namespace string) {
+	pvcs, err := k8sh.Clientset.CoreV1().PersistentVolumeClaims(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		logger.Infof("faied to list pvs. %+v", err)
```

Comment:
typo: `faied` (this typo appears multiple places)

also, the resource being listed is PVCs, not PVs in this function.